src/components/CampaignVisualisations/index.js

```diff
@@ -23,16 +21,8 @@ export default ({ visualisations }) => {
             <CampainVisualisation
               key={index}
               index={index}
-              currentCount={
```

Comment:
Hier hab ich nicht ganz verstanden, wozu das mal gedacht war. currentCounts war in jedem Fall undefined bei mir, daher dann auch der Balken auf null.

Wenn ich es auf true setze, wird die korrekte Zahl angezeigt. Vielleicht ist das aber nicht ganz optimal.